install.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
-CSDIR="$HOME/.config/geany/colorschemes/"
-echo "Installing themes into \`$CSDIR'..."
+
+CSDIR="${XDG_CONFIG_HOME:-$HOME/.config}/geany/colorschemes/"
+echo "Installing themes into '$CSDIR'..."
 mkdir -p "$CSDIR"
-for SCHEME in `ls colorschemes/*.conf`
-do
-  BNAME=`basename "$SCHEME"`
-  echo " => $BNAME"
+
+cd 'colorschemes'
+for SCHEME in *.conf; do
+  echo " => $SCHEME"
```

Comment:
why not simply `for SCHEME in colorschemes/*.conf`?  Also, if you really want to avoid the `basename` call, just use `${SCHEME##*/}` as you don't need of the `basename` fanciness.

Reply:
good point

Reply:
Isn't the ${{##} thing some kind of Bash-ism?

Reply:
> Isn't the ${{##} thing some kind of Bash-ism?

I'm [pretty sure it's POSIX](https://pubs.opengroup.org/onlinepubs/007904875/utilities/xcu_chap02.html#tag_02_06_02) or [at least quite common](https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.osdevice/korn_shell_parmsub1.htm).